src/core/tests/CMakeLists.txt

```diff
@@ -13,8 +13,6 @@ set(OV_ONNX_NAMESPACE ngraph_onnx)
 
 if(ENABLE_TESTS)
     add_definitions("-DTEST_FILES=\"${TEST_MODEL_ZOO}/core/files\"")
-    add_subdirectory(util)
-    add_subdirectory(engines_util)
```

Comment:
If I remember right is used only inside the ngraph tests. Why do we need to move it to common part?

Reply:
They are used in frontend tests as well. So I moved these libs to common location as first step of frontend test movement